CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ This section is for maintaining a changelog for all breaking changes for the cli
 
 ### Fixed
 - Fix version and build ([#254](https://github.com/opensearch-project/opensearch-java/pull/254))
+- Fix integer overflow for variables in indices stats response ([#959](https://github.com/opensearch-project/opensearch-java/pull/960))
```

Comment:
The number #959 doesn't match the link #960, should be 960 everywhere.

Reply:
pushed